.JuliaFormatter.toml

```diff
@@ -0,0 +1,8 @@
+# Configuration file for JuliaFormatter.jl
+# For more information, see: https://domluna.github.io/JuliaFormatter.jl/stable/config/
```

Comment:
I hope we can someday put all these config things into the Project.toml, would clean up nicely.

Reply:
Yeah that'll be nice :)